test/integration/master/synthetic_master_test.go

```diff
@@ -748,7 +748,7 @@ func TestUpdateNodeObjects(t *testing.T) {
 							Reason: "bar",
 						},
 					}
-				case i%4 == 1:
+				case i%4 == 2:
```

Comment:
`==2` is really what we want here? This test is hard to follow.

Reply:
So should I remove this 'case i%4 == 1 ' or else?

Reply:
I don't know :) This test needs some higher level comments to explain what the goal is.